samcli/commands/exceptions.py

```diff
@@ -78,3 +78,9 @@ class AppPipelineTemplateMetadataException(UserException):
     """
     Exception class when SAM is not able to parse the "metadata.json" file located in the SAM pipeline templates
     """
+
+
+class InvalidInitOptionException(UserException):
```

Comment:
Where do we use this exception?

Reply:
Here: https://github.com/aws/aws-sam-cli/pull/2958/files#diff-3ec414e5c1ea9746347f7336f0308441a1031df7076194c96ab9ab541f7660d3R103 (diff was collapsed)